data/brands/amenity/vending_machine.json

```
@@ -354,8 +354,7 @@
       "locationSet": {"include": ["it"]},
```

It looks like this brand was collected from the [planet file](https://taginfo.openstreetmap.org/search?q=Distributore+di+acqua+Aqvagold#values). A `matchName` should be added to prevent this from being automatically readded and to match this entry to existing features with this name.
```suggestion
      "locationSet": {"include": ["it"]},
      "matchNames": ["Distributore di acqua Aqvagold"],
```